manifests/init.pp

```diff
@@ -3,17 +3,91 @@
 # Install, configure and start dns service
 #
 # === Parameters:
+# $namedconf_path::        Path of the named config
+#                          type:Stdlib::Absolutepath
+#
+# $dnsdir::                Directory holding the named configs
+#                          type:Stdlib::Absolutepath
+#
+# $dns_server_package::    Name of the package to install
+#                          type:String
+#
+# $rndckeypath::           Path of the RNDC key
+#                          type:Stdlib::Absolutepath
+#
+# $optionspath::           Path of the named options
+#                          type:Stdlib::Absolutepath
+#
+# $publicviewpath::        Path of the config file holding all the zones
+#                          type:Stdlib::Absolutepath
+#
+# $vardir::                Directory holding the variable or working files
+#                          type:Stdlib::Absolutepath
+#
+# $namedservicename::      Name of the service
+#                          type:String
+#
+# $zonefilepath::          Directory containing zone files
+#                          type:Stdlib::Absolutepath
+#
+# $localzonepath::         File holding local zones like RFC1912 or RFC1918 files.
+#                          type:Optional[Stdlib::Absolutepath]
+#
+# $forward::               The forward option
+#                          type:Optional[String]
+#
+# $forwarders::            The forwarders option
+#                          type:Array[String]
```

Comment:
According to http://www.zytrax.com/books/dns/ch7/queries.html#forwarders they can also include a port so simply forcing it to an IP is not the correct thing to do.